lib/msf/core/db_manager/import/nuclei.rb

```diff
@@ -0,0 +1,76 @@
+module Msf::DBManager::Import::Nuclei
+  #
+  # Imports Nuclei scan results in JSON format.
+  #
+  def import_nuclei_json(args = {}, &block)
+    wspace = Msf::Util::DBManager.process_opts_workspace(args, framework).name
+    bl = validate_ips(args[:blacklist]) ? args[:blacklist].split : []
+
+    args[:data].each_line do |line|
+      next if line.strip.blank?
+
+      data = JSON.parse(line)
+
+      ip = data['ip']
```

Comment:
Since my exported JSON file starts with `[{"template":`, this breaks. I add to do this:
```ruby
      data = JSON.parse(line)[0]
```
which is probably not a real fix.

Reply:
> which is probably not a real fix.

Correct. This PR now supports both JSONL and JSON formats. See https://github.com/rapid7/metasploit-framework/pull/17821#discussion_r1161616149 for context.